client/landing/gutenboarding/index.tsx

```diff
@@ -2,37 +2,115 @@
  * External dependencies
  */
 import '@automattic/calypso-polyfills';
+import { setLocaleData } from '@wordpress/i18n';
+import { I18nProvider } from '@automattic/react-i18n';
+import { getLanguageFile } from '../../lib/i18n-utils/switch-locale';
 import React from 'react';
 import ReactDom from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import config from '../../config';
+import { subscribe, select } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import { Gutenboard } from './gutenboard';
 import { setupWpDataDebug } from './devtools';
 import accessibleFocus from 'lib/accessible-focus';
+import { USER_STORE } from './stores/user';
+
 /**
  * Style dependencies
  */
 import 'assets/stylesheets/gutenboarding.scss';
 import 'components/environment-badge/style.scss';
 
-window.AppBoot = () => {
+const DEFAULT_LOCALE_SLUG: string = config( 'i18n_default_locale_slug' );
+
+type User = import('@automattic/data-stores').User.CurrentUser;
```

Comment:
There are types that don't seem to belong to any app or component. Time for a `calypso-types` package?

- User
- Site
- …

Reply:
> Time for a calypso-types package?

Yes, I agree, you saw I'd added types that I called `UserFactory` and `UserData` returned by `client/lib/user/user.js` which are different from this `User` type defined by data stores... It would be nice to get them all clearly defined. And I think a `calypso-types` package would allow both our `client` code, our `packages`, and any other apps to reuse that sweet type information.